JoinColony/colonyCDapp
# Feat: Implement Multi-Sig extension was uninstalled UI

## Description

- Implement the UI in the action sidebar which should be shown to users when they are accessing a multi sig action _after_ the multi sig extension has been uninstalled.
- Also implement this UI for Reputation Voting actions, since it wasn't there already.

[Link to designs here](https://www.figma.com/design/5V8pr7iMwXsT9L3VAZsmUt/Colony-v5?node-id=22857-221890&t=Eq8J9IE9YnZuNQ7r-4)

## Testing

* Install the multi sig extension
* Create a mint tokens multi sig action
* Uninstall the multi sig extension
* Go back to the mint token action, and check that you have a message in the sidebar explaining that the extension is uninstalled.
<img width="1076" alt="Screenshot 2024-07-18 at 16 23 05" src="https://github.com/user-attachments/assets/6478334d-5f24-4bf7-9d6a-3e26b975ff67">

* Repeat the above steps, but this time for Reputation Voting, not Multi Sig
<img width="1076" alt="Screenshot 2024-07-18 at 16 23 13" src="https://github.com/user-attachments/assets/d8a09aa0-5bb3-496b-b846-90cef42958ff">

## Diffs

**New stuff** ✨

* New UninstalledMessage component, which is flexible and can be used for any extension.

**Changes** 🏗

* Ensure invalid motion statuses are displayed correctly.

Resolves #2712
